lib/Common/Memory/CustomHeap.cpp

```diff
@@ -439,12 +439,12 @@ Allocation* Heap<TAlloc, TPreReservedAlloc>::AllocLargeObject(size_t bytes, usho
     size_t resultBytes = VirtualQueryEx(this->processHandle, address, &memBasicInfo, sizeof(memBasicInfo));
     if (resultBytes == 0)
     {
-        if (this->processHandle != GetCurrentProcess())
```

Comment:
we didn't want to assert for oop jit, because it might be that runtime died, so here we threw

---
In reply to: [104337250](https://github.com/Microsoft/ChakraCore/pull/2619#discussion_r104337250) [](ancestors = 104337250)